libs/application/templates/general-fishing-license/src/lib/GeneralFishingLicenseTemplate.ts

```diff
@@ -0,0 +1,193 @@
+import {
+  Application,
+  ApplicationConfigurations,
+  ApplicationContext,
+  ApplicationRole,
+  ApplicationStateSchema,
+  ApplicationTemplate,
+  ApplicationTypes,
+  DefaultEvents,
+} from '@island.is/application/core'
+import { Events, States, Roles } from '../constants'
+import { GeneralFishingLicenseSchema } from './dataSchema'
+import { application } from './messages'
+import { ApiActions } from '../shared'
+
+const pruneAtMidnight = () => {
+  const date = new Date()
+  const utcDate = new Date(date.toUTCString()) // In case user is not on GMT
+  const midnightDate = new Date(date.toUTCString())
+  midnightDate.setHours(23, 59, 59)
+  const timeToPrune = midnightDate.getTime() - utcDate.getTime()
+  return {
+    shouldBeListed: true,
+    shouldBePruned: true,
+    whenToPrune: timeToPrune,
+  }
+}
+
+const GeneralFishingLicenseTemplate: ApplicationTemplate<
+  ApplicationContext,
+  ApplicationStateSchema<Events>,
+  Events
+> = {
+  type: ApplicationTypes.GENERAL_FISHING_LICENSE,
+  name: application.general.name,
+  institution: application.general.institutionName,
+  readyForProduction: false,
+  translationNamespaces: [
+    ApplicationConfigurations.GeneralFishingLicense.translation,
+  ],
+  dataSchema: GeneralFishingLicenseSchema,
+  stateMachineConfig: {
+    initial: States.PREREQUISITES,
+    states: {
+      [States.PREREQUISITES]: {
+        meta: {
+          name: application.general.name.defaultMessage,
+          progress: 0.1,
+          lifecycle: pruneAtMidnight(),
```

Comment:
should not be listed and prune in 5 mins or so.

Reply:
Fixed